src/applications/vaos/sass/vaos.scss

```diff
@@ -12,6 +12,22 @@
   padding: 14px 0 0 17px;
 }
 
+.vaos-appt-list__icon {
+  &.vaos-appt-list__icon--type-history {
+    background-color: #DBE4EF;
```

Comment:
Where did this color come from? We might want to pick the closest one in https://design.va.gov/design/color-palette, or ask Ryan.